rhobs/alerting/data_plane/prometheus.exporter_alerts.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  name: konflux-exporter-alerting
+  labels:
+    tenant: rhtap
+spec:
+  groups:
+  - name: exporter_alerts
+    interval: 5m
+    rules:
+    - alert: KonfluxUpAlert
+      expr: absent_over_time(konflux_up{namespace="appstudio-grafana-datasource-exporter"}[5m]) == 1
+      for: 5m
+      labels:
+        severity: warning
+      annotations:
+        summary: Exporter metric 'konflux_up' is missing.
+        description: >-
+          The Prometheus exporter metric 'konflux_up' for namespace
+          "appstudio-grafana-datasource-exporter" on cluster
+          {{ $labels.source_cluster }} is missing for more than 30 minutes.
+        alert_routing_key: 'appstudio-grafana-datasource-exporter'
```

Comment:
@avi-biton would that mean creating a new entry for routing this new key?
Maybe we can use our existing key for routing for now.

Reply:
Looking at the alertmanager [config](https://gitlab.cee.redhat.com/service/app-interface/-/blob/d35086d821680821d3f6b9827e41b45a6bd16da8/resources/observability/alertmanager/alertmanager-instance.secret.yaml#L3555), there's no condition with this routing key but there is one for `appstudio-grafana`. I think we can use that instead.